riscv/processor.cc

```diff
@@ -765,6 +765,7 @@ void processor_t::enter_debug_mode(uint8_t cause)
   state.debug_mode = true;
   state.dcsr->write_cause_and_prv(cause, state.prv, state.v);
   set_privilege(PRV_M);
+  set_virt(false);
```

Comment:
Sadly, this commit will be a NOP, because privilege is already PRV_M so this change will [get blocked](https://github.com/riscv-software-src/riscv-isa-sim/blob/ebf9a94af86f7c56e24d12f104ef9e983e836f99/riscv/processor.cc#L748-L749).

I will fix this in my rework.

Reply:
Ah, right... okay, thanks again.